incubator/validator-protobuf.spec/src/main/scripts/io/aklivity/zilla/specs/validator/protobuf/config/validator.yaml

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright 2021-2023 Aklivity Inc
+#
+# Licensed under the Aklivity Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+---
+name: test
+catalogs:
+  test0:
+    type: test
+    options:
+      schema: |
+        syntax = "proto3";
+        message example 
+        {
+          string id = 1;
+          string status = 2;
+        }
+bindings:
+  test:
+    kind: server
+    type: test
+    options:
+      value:
+        type: protobuf
+        catalog:
+          catalog0:
+            - subject: test0
+              version: latest
+              record: SimpleMessage
```

Comment:
Does `SimpleMessage` not need to align with one of the message names, like `example`?
Also, perhaps it needs a package prefix too?